web/src/main/java/edu/softserve/zoo/web/security/JwtRememberMeServices.java

```diff
@@ -0,0 +1,112 @@
+package edu.softserve.zoo.web.security;
+
+import edu.softserve.zoo.model.Employee;
+import edu.softserve.zoo.service.EmployeeService;
+import edu.softserve.zoo.service.security.AuthUserDetails;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.authentication.RememberMeServices;
+import org.springframework.security.web.authentication.logout.LogoutHandler;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Implements JWT remember-me functionality.
+ *
+ * @author Ilya Doroshenko
+ */
+@Component
+public class JwtRememberMeServices implements RememberMeServices, LogoutHandler {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(JwtRememberMeServices.class);
+
+    @Autowired
+    private UserDetailsService userDetailsService;
+
+    @Autowired
+    private EmployeeService employeeService;
+
+    @Autowired
+    private JwtUtils jwtUtils;
+
+    @Value("${token.header}")
+    private String tokenHeaderName;
+
+    /**
+     * Called when security filter tries to authenticate request via its token.
+     */
+    @Override
+    public Authentication autoLogin(HttpServletRequest request, HttpServletResponse response) {
+        String authToken = request.getHeader(tokenHeaderName);
+        String username = jwtUtils.getUsernameFromToken(authToken);
+
+        if (username != null && SecurityContextHolder.getContext().getAuthentication() == null) {
```

Comment:
You are still trying to ignore what I say to you. Where's login for user which already logged in?